CMakeLists.txt

```diff
@@ -354,10 +358,43 @@ else ()
   endif()
 endif ()
 
+add_executable(cont_test
```

Comment:
So as mentioned earlier today we may want to separate these changes out into just the new verifier stuff and then a separate repo which is for the experiment code